### PR TITLE
Add gallery.

### PR DESCRIPTION
**Pull request by KonovalovaVV**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,10 +118,51 @@ <h3 class="feature-card__lead lead">Civamus vesti</h3>
...
+        </button>
+        <button class="circle-button circle-button--small">
+          <i class="circle-button__icon fas fa-arrow-right"></i>
+        </button>
+        <hr class="separator separator--right">
+      </div>
+
+      <div class="gallery">
+        <div class="gallery__item">
+          <img src="img/gallery1.png" alt="">
```
Comment (MaksHladki) [inline index.html:148]: Please, re-heck styles. I think we have to add width: 100% and height 100% for this image.

File: index.html [hunk 2]
```diff
@@ -119,10 +118,51 @@ <h3 class="feature-card__lead lead">Civamus vesti</h3>
...
+        <div class="gallery__item">
+          <img src="img/gallery3.png" alt="">
+        </div>
+        <div class="gallery__item">
+          <img src="img/gallery4.png" alt="">
+        </div>
+      </div>
+
+      <div class="gallery-button">
+        <button class="orange-button">View portfolio</button>
```
Comment (MaksHladki) [inline index.html:162]: Add border: none.

File: index.html [hunk 3]
```diff
@@ -119,10 +118,51 @@ <h3 class="feature-card__lead lead">Civamus vesti</h3>
...
+          <h2 class="header-label header-label--dark">Cras ornare tristique elit</h2>
+          <p class="header-descripton header-descripton--dark">Donec odio uisque volutpat mattis eros ullam malesuada
+            erat
+            ut turpis. </p>
         </div>
       </div>
 
+      <div class="arrow-separator">
+        <hr class="separator separator--left">
+        <button class="circle-button circle-button--small">
```
Comment (MaksHladki) [inline index.html:137]: Add aria labels "Go to next/previous image"

File: index.html [hunk 4]
```diff
@@ -100,7 +99,7 @@ <h3 class="feature-card__lead lead">Aliquam tinci</h3>
 
           <section class="feature-card">
```
Comment (MaksHladki) [inline index.html:100]: Please, re-check styles.
image

File: index.html (hunk 3 above)
Comment (KonovalovaVV) [reply index.html:137]: Ok.

File: index.html (hunk 1 above)
Comment (KonovalovaVV) [reply index.html:148]: Ok.

File: index.html (hunk 4 above)
Comment (KonovalovaVV) [reply index.html:100]: Ok.

File: index.html (hunk 1 above)
Comment (MaksHladki) [reply index.html:148]: Revert your changes and fix the issue.

**Pull request merged**